docs/release-notes/2.42.md

```diff
@@ -0,0 +1,122 @@
+
+- [PMM-13123](https://perconadev.atlassian.net/browse/PMM-13123) - Enhanced the **Query Response Time Distribution** graphs in the **MySQL Query Response Time Details** dashboard to provide more granular insights into query performance. Previously, the graphs did not include buckets for queries with response times less than 100 milliseconds, which is where many queries typically fall. This limitation made it difficult to analyze the performance of sub-100ms queries, especially those in the sub-1ms range.
+
+- [PMM-13075](https://perconadev.atlassian.net/browse/PMM-13075) - Added support for Ubuntu 24.04 support.
+
+- [PMM-12896](https://perconadev.atlassian.net/browse/PMM-PMM-12896), [PMM-12895](https://perconadev.atlassian.net/browse/PMM-12895), [PMM-12971](https://perconadev.atlassian.net/browse/PMM-12971) - Added per-database metrics collection and control of parameters, enabling more granular data gathering without overusing connections.
+
+- [PMM-12994](https://perconadev.atlassian.net/browse/PMM-12994) - Added labels to the `pg_replication_slot_slot_is_active` and `slot_current_wal_lsn` metrics in the postgres_exporter to identify the replication type (logical or physical) and the plugin used for the slot.
+
+- [PMM-12753](https://perconadev.atlassian.net/browse/PMM-12753) - Added experimental versions of the MongoDB Cluster Summary and ReplicaSet dashboards with a streamlined design and simplified navigation, focusing on essential metrics for monitoring MongoDB performance.
+
+- [PMM-11583](https://perconadev.atlassian.net/browse/PMM-11583) - Added support for the `innodb_redo_log_capacity` variable introduced in MySQL 8.0, ensuring accurate and consistent data representation in the InnoDB Logging graphs.
+
+### Fixed issues
+
+- [PMM-12349](https://perconadev.atlassian.net/browse/PMM-12349) - Fixed an issue in the MongoDB RepSet Summary dashboard where incorrect data was displayed when a node in the replica set was down, ensuring that the dashboard accurately reflects the status and version information of the nodes even when they are unavailable.
+
+- [PMM-12522](https://perconadev.atlassian.net/browse/PMM-12522) - Resolved an issue where high resolution collection of MongoDB sharding information caused timeouts.
+
+- [PMM-12880](https://perconadev.atlassian.net/browse/PMM-12880) - Fixed an issue where the `pmm-admin --tls-skip-verify` flag did not work correctly when using x509 authentication for MySQL, ensuring that the flag now properly skips certificate verification when connecting to MySQL instances with self-signed or untrusted certificates.
+
+- [PMM-12962](https://perconadev.atlassian.net/browse/PMM-12962) -  Updated the balancer metrics in MongoDB exporter to accurately calculate and report the `mongodb_mongos_sharding_balancer_enabled` metric, providing a correct indication of whether the balancer is currently enabled or disabled in the sharded cluster.
```

Comment:
`mongodb_mongos_sharding_chunks_is_balanced` metric was renamed to `mongodb_mongos_sharding_chunks_is_balancer_running`. We need to recommend to use new metric if user used it in alerts or dashboards.

Reply:
Thanks @BupycHuk! I have updated the entry with the text below. Let me know if something needs changing. 
Also, do we record a What's New section of the API docs for such info? 

```
- [PMM-12962](https://perconadev.atlassian.net/browse/PMM-12962) -  Updated the balancer metrics in MongoDB exporter to accurately calculate and report the `mongodb_mongos_sharding_balancer_enabled` metric, providing a correct indication of whether the balancer is currently enabled or disabled in the sharded cluster.
Additionally, the `mongodb_mongos_sharding_chunks_is_balanced` metric has been renamed to `mongodb_mongos_sharding_chunks_is_balancer_running`. Update any alerts or dashboards using the old metric name to ensure they continue working correctly.
```

Reply:
Not sure if we need separate section or not, but since we mention it in release notes I think we need to provide a link how to do that in case it's only in API.